include/onnxruntime/core/framework/execution_provider.h

```diff
@@ -326,6 +326,14 @@ class IExecutionProvider {
    */
   virtual std::vector<AllocatorPtr> CreatePreferredAllocators() { return std::vector<AllocatorPtr>(); };
 
+  /**
+   * Get the array of pointers for EPContext nodes
```

Comment:
can you add more description of this optional interface?
EP developers need to know whether they should implement it or not.